frontend/src/app/modules/lessons/lessons-page/lessons-page.component.ts

```diff
@@ -89,13 +89,15 @@ export class LessonsPageComponent implements OnInit, OnChanges {
     }
 
     private mapLessons(response: ILesson[]): Lesson[] {
+        const offset = new Date().getTimezoneOffset();
+
         return response.map((lesson) => ({
             id: lesson.id,
             imgPath: lesson.mediaPath,
             videoId: lesson.youtubeVideoId,
             zoomLink: lesson.zoomMeetingLink,
             title: lesson.name,
-            time: moment(lesson.startAt).format('hh.mm'),
+            time: moment(lesson.startAt).add(Math.abs(offset), 'minutes').format('HH.mm'),
```

Comment:
Why we cannot just use Date as a time type?

Reply:
I don't know, I thought that If it has string type, then it will be fine if I will leave it like that. My fix was related to displaying right time)
If we need to change it, I think I can do it. So should I change it?